tutorials/commands/index.rst

```diff
@@ -0,0 +1,20 @@
+.. _command-tutorials:
+
+Command Tutorials
+=================
+
+Welcome to the Command Tutorials! These tutorials should help you to map specific Flux commands
+with your use case, and then see detailed usage. 
+
+ - ``flux proxy`` (:ref:`ssh-across-clusters`): "Send commands to a flux instance across clusters using ssh"
+ - ``job-submit`` (:ref:`job-submit`): "Submit a job in a Flux instance"
```

Comment:
Should this be `flux mini submit` ? I think here we want to start to help to make associations between the actual command to be run and the use case.

Reply:
If I understand what you mean, I think the other PR needs to be merged first. This PR just contains the other PR's commits which is why the file is new.

Reply:
Yep, that is what I was trying to say, poorly.